algorithms/gr-dij/java/Dijkstra.java

```diff
@@ -0,0 +1,89 @@
+import java.lang.*;
+
+class Dijkstra {
+    // A utility function to find the vertex with minimum distance value,
+    // from the set of vertices not yet included in shortest path tree
+    static final int V = 9;
+
+    int minDistance(int dist[], Boolean sptSet[]) {
+        // Initialize min value
+        int min = Integer.MAX_VALUE, min_index = -1;
+
+        for (int v = 0; v < V; v++)
+            if (sptSet[v] == false && dist[v] <= min) {
+                min = dist[v];
+                min_index = v;
+            }
+
+        return min_index;
+    }
+
+    // A utility function to print the constructed distance array
+    void printSolution(int dist[], int n) {
+        System.out.println("Vertex Distance from Source");
+        for (int i = 0; i < V; i++)
+            System.out.println(i + " tt " + dist[i]);
+    }
+
+    // Function that implements Dijkstra's single source shortest path
+    // algorithm for a graph represented using adjacency matrix
+    // representation
+    void dijkstra(int graph[][], int src) {
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6333211)